opm/autodiff/SimulatorFullyImplicitBlackoilOutput.hpp

```diff
@@ -406,6 +406,13 @@ namespace Opm
                          SimulationDataContainer& simulatorstate,
                          WellStateFullyImplicitBlackoil& wellstate)
     {
+        std::map<std::string, UnitSystem::measure> solution_keys {{"PRESSURE" , UnitSystem::measure::pressure},
+                                                                  {"SWAT" , UnitSystem::measure::identity},
+                                                                  {"SGAS" , UnitSystem::measure::identity},
+                                                                  {"TEMP" , UnitSystem::measure::temperature},
+                                                                  {"RS" , UnitSystem::measure::identity},
```

Comment:
If the measure is used to find the unit conversion for input, this will probably be wrong. While its *dimension* is unity (i.e. it's dimensionless), in field units the numerical value is not 1 (1000 ft^3/barrel, and the inverse for RV). If this is *not* an error, what is this information used for?